vuepress/docs/docs/server-codegen.md

```diff
@@ -62,5 +62,79 @@ And you'll get a case class which looks something like this:
 case class MyType(myLazyField: zio.UIO[String], myField: String)
 ```
 
-When implementing this, `myLazyField` will only be avaluated if the client requested it in the query
+When implementing this, `myLazyField` will only be evaluated if the client requested it in the query
 
+### Newtype declaration
+
+`@Newtype` directive in caliban allows you to wrap your GraphQL fields into statically
```

Comment:
```suggestion
The `@newtype` directive in caliban allows you to wrap your GraphQL fields into statically
```

Fixed the casing.

Also, do you mind adding the docs change in another PR? We push docs on merge, but since this won't be available until the next release, if it's in a separate PR I can merge it when I do the release 🙏